packages/hop-node/src/db/TransfersDb.ts

```diff
@@ -105,6 +160,7 @@ class TransfersDb extends TimestampedKeysDb<Transfer> {
     promises.push(this._update(transferId, transfer).then(async () => {
       const entry = await this.getById(transferId)
       logger.debug(`updated db transfer item. ${JSON.stringify(entry)}`)
+      await this.updateIncompleteItem(entry)
```

Comment:
I think a lookup can be done to see if the item 'incomplete' status has changed or not to determine if it should perform the sub db write or not. This would be similar to what we already do with the timestamped key update. The long term solution would be to index every property of the transfer item which would achieve what you're thinking in terms of looking up if specific properties are missing

Reply:
I understand. Thanks for the explanation. I think this is good as-is.